apps/ledger-live-desktop/src/renderer/reducers/settings.ts

```diff
@@ -265,12 +268,6 @@ const handlers: SettingsHandlers = {
     };
   },
   FETCH_SETTINGS: (state, { payload: settings }) => {
-    if (
-      settings.counterValue &&
-      !supportedCountervalues.find(({ currency }) => currency.ticker === settings.counterValue)
-    ) {
-      settings.counterValue = INITIAL_STATE.counterValue;
```

Comment:
this mechanism that was in place was to secure the fact a user, coming with a Countervalues from the past app state, may no longer have that countervalue available (supported) anymore by us.
In the new implementation, I don't see where the same mecanism is preserved. 

A way would be to move this previous logic into the handling of `SET_SUPPORTED_COUNTER_VALUES ` that receives the fetched list and the logic can now possibly re-set the counterValue to a default in case user's countervalue is no longer possible.

Reply:
From what I saw that mechanism is actually handled elsewhere, by the way we get the active counter value. I've tested by removing the selected one from the response locally and it falls back as expected.